src/main/java/seedu/address/model/ItemModelManager.java

```diff
@@ -523,6 +514,58 @@ private VisualizeList getNextTask() {
         return result;
     }
 
+    /**
+     * Turns off the priority mode.
+     */
+    private void toggleOffPriorityMode() {
+        if (timer != null) {
+            timer.cancel();
```

Comment:
Do you want to consider putting this method in the shutdown method of javaFX as well?

Reply:
Done!